Makefile

```
@@ -33,21 +33,21 @@ build-latest:
 
 build-testing:
 	$(MAKE) build TAG=testing \
-	    PROTOCOL_VERSION_DEFAULT=22 \
-		XDR_REF=v22.0.0 \
-		CORE_REF=v22.3.0 \
-		HORIZON_REF=horizon-v22.0.3 \
-		STELLAR_RPC_REF=v22.1.2 \
-		FRIENDBOT_REF=horizon-v22.0.3
+		PROTOCOL_VERSION_DEFAULT=23 \
+		XDR_REF=v23.0.0-rc.1 \
```

According to https://github.com/stellar/rs-stellar-xdr/releases, this should be .2

```suggestion
		XDR_REF=v23.0.0-rc.2 \
```

---

Thanks, updated.